### PR TITLE
KAA-975 Review disabled tests

### PR DESCRIPTION
**Pull request by Dubland**
/cc @rasendubi @GamovCoder

### CODE REVIEW

File: client/client-multi/client-c/listfiles/UnitTest.cmake [hunk 1]
```diff
@@ -224,3 +224,25 @@ kaa_add_unit_test(NAME test_kaa_reallocation
...
+#        test/test_kaa_bootstrap_manager.c
+#        DEPENDS
+#        kaac ${OPENSSL_LIBRARIES})
+
+# KAA-988
+#kaa_add_unit_test(NAME test_kaa_channel_manager
+#        SOURCES
+#        test/test_kaa_channel_manager.c
+#        DEPENDS
+#        kaac ${OPENSSL_LIBRARIES})
```
Comment (rasendubi) [inline client/client-multi/client-c/listfiles/UnitTest.cmake]: Add newline

File: client/client-multi/client-c/test/test_kaa_bootstrap_manager.c [hunk 2]
```diff
@@ -192,15 +189,15 @@ void test_handle_server_sync(void **state)
...
     kaa_transport_channel_interface_t protocol1_channel;
     test_create_channel_interface(&protocol1_channel, &protocol1_channel_context);
 
-    test_channel_context_t protocol2_channel_context = { protocol2, NULL, SUPPORTED_SERVICES, SUPPORTED_SERVICES_COUNT };
+    test_channel_context_t protocol2_channel_context = { protocol2, NULL, SUPPORTED_SERVICES, SUPPORTED_SERVICES_COUNT, {NULL} };
     kaa_transport_channel_interface_t protocol2_channel;
     test_create_channel_interface(&protocol2_channel, &protocol2_channel_context);
 
-    test_channel_context_t protocol3_channel_context = { protocol3, NULL, SUPPORTED_SERVICES, SUPPORTED_SERVICES_COUNT };
+    test_channel_context_t protocol3_channel_context = { protocol3, NULL, SUPPORTED_SERVICES, SUPPORTED_SERVICES_COUNT, {NULL} };
```
Comment (rasendubi) [inline client/client-multi/client-c/test/test_kaa_bootstrap_manager.c]: The lines are too long. Could you please split them?

File: client/client-multi/client-c/test/test_kaa_channel_manager.c [hunk 3]
```diff
@@ -180,7 +183,7 @@ void test_add_channel(void **state)
 
     kaa_transport_channel_interface_t expected_channel;
     test_create_channel_interface(&expected_channel, &channel_context);
-
+
```
Comment (rasendubi) [inline client/client-multi/client-c/test/test_kaa_channel_manager.c]: trailing spaces

File: client/client-multi/client-c/test/test_kaa_bootstrap_manager.c [hunk 4]
```diff
@@ -132,18 +132,15 @@ void test_create_bootstrap_manager(void **state)
...
+    error_code = kaa_bootstrap_manager_create(NULL, &kaa_context);
     ASSERT_NOT_EQUAL(error_code, KAA_ERR_NONE);
 
-    error_code = kaa_bootstrap_manager_create(&manager, NULL, logger);
+    error_code = kaa_bootstrap_manager_create(&manager, NULL);
     ASSERT_NOT_EQUAL(error_code, KAA_ERR_NONE);
 
-    error_code = kaa_bootstrap_manager_create(&manager, channel_manager, NULL);
+    error_code = kaa_bootstrap_manager_create(&manager, &kaa_context);
     ASSERT_NOT_EQUAL(error_code, KAA_ERR_NONE);
```
Comment (rasendubi) [inline client/client-multi/client-c/test/test_kaa_bootstrap_manager.c]: This should be equal (`assert_int_equal`). All parameters are correct.

**Comment by forGGe** [comment]
Add newline, as @rasendubi  proposed.
Otherwise looks good.

**Comment by rasendubi** [comment]
LGTM

**Pull request merged**